packages/lesswrong/lib/loaders.ts

```diff
@@ -36,7 +36,7 @@ export async function getWithLoader<N extends CollectionNameString>(
         ...baseQuery,
         [groupByField]: {$in: docIDs}
       };
-      const queryResults: ObjectsByCollectionName[N][] = await Utils.Connectors.find(collection, query, projection);
+      const queryResults: ObjectsByCollectionName[N][] = await collection.find(query, projection).fetch();
```

Comment:
Maybe this was already broken but I think I recently ran into a case where passing in the projection as part of the second argument didn't work, while passing it in as the third did.

But it might actually not be a problem; I think the `projection` arg in `getWithLoader` is actually meant to be `options` - it only seems to be used in the `make_editable` `revisions` field resolver, where it passes in `{sort: {editedAt: -1}, limit}`.  So probably we want to change `projection: any=undefined` to `options?: MongoFindOptions<ObjectsByCollectionName[N]> = undefined`, so that it accurately represents what it's passing through.

Reply:
Yep, this analysis is correct; it's actually `options`, misnamed as `projection` (it was misnamed that way both before and after the refactor). Renaming to `options` and adding the type annotation.